File: java/vampire/sample/lang/SampleGetSystemInformations.java
/*
 *  Licensed under the Apache License, Version 2.0 (the "License");
 *  you may not use this file except in compliance with the License.
 *  You may obtain a copy of the License at 
 * 
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 *  Unless required by applicable law or agreed to in writing, software
 *  distributed under the License is distributed on an "AS IS" BASIS,
 *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 *  See the License for the specific language governing permissions and
 *  limitations under the License.
 *  
 *  Copyright © 2020 Sebastian Ritter
 */
package vampire.sample.lang;
import java.lang.System;

public class SampleGetSystemInformations
{
    public static void main(String[] args)
    {
        java.util.Properties p = java.lang.System.getProperties();
        p.list(java.lang.System.out);
    }
}

File: python/src/vampire/sample/lang/SampleGetSystemInformations.py
# SPDX-FileCopyrightText: 2020 - Sebastian Ritter <bastie@users.noreply.github.com>
# SPDX-License-Identifier: LicenseRef-NONE

from java.lang.System import System
from java.lang.Object import Object


class SampleGetSystemInformations (Object):
    
    
    @classmethod
    def main (_ : type):
        p = System.getProperties();
        p.list(System.out);


SampleGetSystemInformations.main()
